src/app/component/auth/components/sign-up/sign-up.component.ts

```diff
@@ -87,7 +86,7 @@ export class SignUpComponent implements OnInit {
   }
 
   private onSubmitError(errors: HttpErrorResponse): void {
-    errors.error.forEach(error => {
+    errors.error.map(error => {
       switch (error.name) {
```

Comment:
very strange switch - it has only one case. should be changed

Reply:
> very strange switch - it has only one case. should be changed

Fixed